client/extensions/woocommerce/components/process-orders-widget/index.js

```diff
@@ -20,15 +20,15 @@ const ProcessOrdersWidget = ( { className, site, orders, currency, ordersRevenue
 			<div>
 				<span>{ orders.length }</span>
 				<span className="process-orders-widget__order-label">
-					{ translate( 'New orders' ) }
+					{ translate( '✨ New orders' ) }
 				</span>
 			</div>
 			<div>
 				<span className="process-orders-widget__revenue-amount">
 					{ formatCurrency( ordersRevenue, currencyValue ) || ordersRevenue }
 				</span>
 				<span className="process-orders-widget__revenue-label">
-					{ translate( 'Revenue' ) }
+					{ translate( '💰 Revenue' ) }
```

Comment:
:disappointed: 30 existing translations will be lost with this change.
If the string must change, the following option has already been translated **19** times:
`translate( 'Revenue Generation' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/dashboard-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).